app/views/activities/_share_button.html.erb

```diff
@@ -1,4 +1,15 @@
-<%= <div id="fb-root"></div>
+<%# <div id="fb-root"></div>
 <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_GB/sdk.js#xfbml=1&version=v15.0" nonce="1WMVtyEy"></script>
 <div class="fa fa-share-alt" data-href="https://developers.facebook.com/docs/plugins/" data-layout="button_count" data-size="small"><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&amp;src=sdkpreparse" class="fb-xfbml-parse-ignore"></a></div> %>
-<span><%= button_to "", "https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&amp;src=sdkpreparse", class:" icons border-0 fa fa-share-alt bg-transparent" %></span>
+<%# <span>button_to "", "https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&amp;src=sdkpreparse", class:" icons border-0 fa fa-share-alt bg-transparent"</span> %>
+
+
+<script>
+document.getElementById('shareBtn').onclick = function() {
```

Comment:
we need to have the id on the share button

From the example. `<div id="shareBtn" class="btn btn-success clearfix">Share Dialog</div>`

Reply:
ah sorry i realised i accidentally deleted the id, have added it back in the latest commit!

Reply:
and i think don't need the meta tags, so i removed those in the application.html.erb too